crates/sui-framework/docs/std/u128.md

```diff
@@ -191,7 +191,7 @@ function can only operate with integers, it is impossible
 to get perfect (or precise) integer square root for some numbers.
 
 Example:
-```
+```move
```

Comment:
These blocks are generated via (see [external-crates/move/crates/move-docgen/src/docgen.rs](https://github.com/MystenLabs/sui/blob/fd84fc1dde7675d81ba3eb5c4fc979e2f17d07f9/external-crates/move/crates/move-docgen/src/docgen.rs#L1)), so we will need to change that code generation rather than these files